pkg/deploy/testdata/happyPathDeployEngineRequest.json

```diff
@@ -47,6 +47,15 @@
       },
       "strategy": {
         "canary": {
+          "trafficManagement": [
```

Comment:
The format here looks wrong... but I'm not sure this is being used anywhere.

Reply:
Should be updated for the DE multi-deployment format